src/elements/content-sidebar/ActivitySidebar.js

```diff
@@ -191,9 +191,17 @@ class ActivitySidebar extends React.PureComponent<Props, State> {
             // need to load the pending item
             this.fetchFeedItems();
         },
-        updateTask: ({ text, id }: { id: string, text: string }): void => {
-            const { file, api } = this.props;
-            api.getFeedAPI(false).updateTask(file, id, text, this.feedSuccessCallback, this.feedErrorCallback);
+        updateTask: (task: TaskUpdatePayload): void => {
+            const { file, api, onTaskUpdate = noop } = this.props;
```

Comment:
Default value for `onTaskUpdate` should live in the `defaultProps` static property.